layouts/about-release-schedule.hbs

```diff
@@ -19,6 +19,13 @@
 
         {{{ release-schedule }}}
 
+        <p>
+          Major Node versions enter "Current" release status for six months, which gives library authors time to add support for them.
+          After six months, odd-numbered releases (9, 11, etc.) become unsupported, and even-numbered releases (10, 12, etc.) move to "Active LTS" status and are ready for general use.
+          "LTS" release status is "long-term support", and guarantees that critical bugs will be fixed for a total of 30 months.
```

Comment:
This isn't always 30 months, I think? Some versions have shorter lifecycles, often due to EoL of an included OpenSSL version.

Reply:
I based this on this:

> Every major version covered by the LTS plan will be actively maintained for a period of 18 months from the date it enters LTS coverage. Following those 18 months of active support, the major version will transition into "maintenance" mode for 12 additional months.

which I found here: https://github.com/nodejs/Release#release-plan

Reply:
Perhaps the solution is to add a "usually" or a "typically" in there. (Same on the source document.)

Reply:
Thanks, I did exactly that.